Source/WebKit/WebProcess/GPU/media/MediaPlayerPrivateRemote.cpp

```diff
@@ -401,15 +447,15 @@ bool MediaPlayerPrivateRemote::seeking() const
 void MediaPlayerPrivateRemote::rateChanged(double rate)
 {
     m_rate = rate;
+    m_currentTimeEstimator.setRate(rate);
```

Comment:
Does it make sense to duplicate m_rate in MediaPlayerPrivateRemote and time estimator?

Reply:
I did so because one is only ever accessed on the main thread, while the other requires a lock.